system/services/rendering/LabelRendererService.cfc

```diff
@@ -61,7 +61,8 @@ component {
 				, eventArguments = args
 			);
 		} else {
-			return args.label ?: args[ $getPresideObjectService().getLabelField( objectname=args.object )] ?: "";
+
+			return args.label ? EncodeForHTML( args.label ) : args[ $getPresideObjectService().getLabelField( objectname=args.object )] ?: "";
```

Comment:
`args.label` is not a boolean. Also I think we should be encoding both when we have `label` and also when we fetch from a non default label field? So:

```cfc
if ( StructKeyExists( args, "label" ) ) {
    return EncodeForHtml( args.label );
}
return EncodeForHtml( args[ $getPresideObjectService().getLabelField( objectname=args.object ) ] ?: "" );
```

Or just wrap the original line in `EncodeForHtml`:

```cfc
return EncodeForHtml( args.label ?: ( args[ $getPresideObjectService().getLabelField( objectname=args.object ) ] ?: "" ) );
```